tests/kernel/install_ltp.pm

```diff
@@ -286,11 +286,16 @@ sub get_default_pkg {
 sub install_from_repo {
     my @pkgs = split(/\s* \s*/, get_var('LTP_PKG', get_default_pkg));
 
-    zypper_call("in --recommends " . join(' ', @pkgs));
+    if (is_transactional) {
+        assert_script_run("transactional-update -n -c pkg install " . join(' ', @pkgs), 90);
```

Comment:
Can't it just be `"transactional-update -n -c pkg install @pkgs"` ?

Reply:
I think the main concern is one universal function to cover installation for both types and don't use if-else logic in the test code on different places.